radixdlt-core/radixdlt/src/test/java/com/radixdlt/client/handler/HighLevelApiHandlerTest.java

```diff
@@ -74,13 +74,12 @@
 import static com.radixdlt.utils.functional.Tuple.tuple;
 
 public class HighLevelApiHandlerTest {
-	private static final byte MAGIC = (byte) 0;
-	private static final String KNOWN_ADDRESS_STRING = "JH1P8f3znbyrDj8F4RWpix7hRkgxqHjdW2fNnKpR3v6ufXnknor";
-	private static final RadixAddress KNOWN_ADDRESS = RadixAddress.from(KNOWN_ADDRESS_STRING);
-
-	private static final RadixAddress V1 = new RadixAddress(MAGIC, ECKeyPair.generateNew().getPublicKey());
-	private static final RadixAddress V2 = new RadixAddress(MAGIC, ECKeyPair.generateNew().getPublicKey());
-	private static final RadixAddress V3 = new RadixAddress(MAGIC, ECKeyPair.generateNew().getPublicKey());
+	private static final ECPublicKey PUB_KEY = ECKeyPair.generateNew().getPublicKey();
+	private static final REAddr ACCOUNT_ADDR = REAddr.ofPubKeyAccount(PUB_KEY);
```

Comment:
slighlt confusing names here, the actual AccountAddress is called `ADDRESS` and the ("abstract") REAddr is called `ACCOUNT_ADDR`.  Not important but I would slightly change that?